code/modules/SCP/SCPs/SCP-457.dm

```diff
@@ -11,9 +11,9 @@
 	SCP = /datum/scp/scp_457
 	status_flags = NO_ANTAG
 	var/door_cooldown
-	health = 500
-	maxHealth = 500
-	var/next_emote = -1
+	health = 700
+	maxHealth = 700
+	var/funnymode = FALSE //this makes 457 better (for events and shit)
```

Comment:
Unused variable. I'd also suggest adding variables for individual abilities of the SCP, so admins can edit whatever they want.

Reply:
got it

Reply:
done